ppafm/ocl/cl/FF.cl

```diff
@@ -619,18 +748,18 @@ __kernel void gradPotential(
     int jn = j < nGrid.y - 1 ? ind + nGrid.z : ind - (nGrid.y - 1) * nGrid.z;
```

Comment:
this is idex wraping due to PBC? maybe put there coment that this grad works in PBC ?

Reply:
Yes, it's for PBC. I added comments both to the OCL kernel and the Python wrapper method.